examples/ransomware_detection/README.md

```diff
@@ -43,7 +43,7 @@ docker run --rm -ti --gpus=all -p8000:8000 -p8001:8001 -p8002:8002 -v $PWD/model
    tritonserver --model-repository=/models/triton-model-repo \
                 --exit-on-error=false \
                 --model-control-mode=explicit \
-                --load-model ransomw-model-short-rf
+                --load-model ransomware_model_tl
```

Comment:
All of our models use hyphens instead of underscores. Can you rename the model `ransomware-model-tl`?